sources/naturalearth.json

```diff
@@ -3,9 +3,11 @@
 	"fullname": "Natural Earth",
 	"name": "naturalearth",
 	"prefix": "ne",
-	"key":"id",
+	"key": "id",
```

Comment:
Actually, while there isn't an `id` explicitly, sometimes Natural Earth does have sable IDs for things, but they're not stored in an ID column. 

In https://github.com/whosonfirst-data/whosonfirst-data/issues/103 and the related PR, sometimes the concorance will be:

- `"ne:sov_a3":"FR1"`
- `"ne:adm0_a3":"FRA"`

So would `sov_a3` and `adm0_a3` need to be listed in the key here? Is key allowed to be a list?

(I know, I know, Natural Earth should have stable IDs, but it doesn't today.)